task6.cpp

```diff
@@ -0,0 +1,59 @@
+#include <iostream>
+using namespace std;
+struct Point {
+    int x, y;
+};
+void readPoint(Point& point) {
+    cout << "X: ";
+    cin >> point.x;
+    cout << endl;
+    cout << "Y: ";
+    cin >> point.y;
+    cout << endl;
+}
+void printPoint(const Point& point) {
+    cout << "X: " << point.x << endl;
+    cout << "Y: " << point.y << endl;
+}
+double distanceFromCenter(const Point& point) {
+    return sqrt(point.x * point.x + point.y * point.y);
+}
+double distanceBetweenPoint(const Point& point1, const Point& point2) {
+    int dx = point1.x - point2.x;
+    int dy = point1.y - point2.y;
+    return sqrt(dx * dx + dy * dy);
+}
+int quadrant(const Point& point) {
+    if (point.x >= 0 && point.y >= 0) {
+        return 1;
+    }
+    else if (point.x <= 0 && point.y >= 0) {
+        return 2;
+    }
+    else if (point.x <= 0 && point.y <= 0) {
+        return 3;
+    }
+    else {
+        return 4;
+    }
+}
+bool doesItBelong(Point& point, double& r) {
+    if (distanceFromCenter(point) - r < 0) {
+        return false;
+    }
+    else true;
+}
```

Comment:
One liner => directly return
Please, never write code of type 
if(sth) 
return true 
else
return false